backend/api/CourseApi/index.js

```diff
@@ -48,7 +49,8 @@ router.get('/public', catchAsync(async (request, response) => {
     amountofAllCoursesQuery = amountofAllCoursesQuery.andWhere('course_category_id', courseCategoryId);
   }
 
-  const amountOfAllCourses = await amountofAllCoursesQuery;
+  let amountOfAllCourses = await amountofAllCoursesQuery;
+  amountOfAllCourses = amountOfAllCourses[0].amountOfPublicCourses;
```

Comment:
👍Lgtm
I'll merge this now, but for the future, - we can do `const amountOfAllCourses = (await amountofAllCoursesQuery)[0].amountOfPublicCourses;` whenever it looks better.

Reply:
Sure. Thanks :)